site/src/templates/__tests__/standard-articles.js

```diff
@@ -198,4 +199,40 @@ describe("StandardArticle", () => {
 
     expect(articleInstance.hasRoles(post)).toBeFalsy()
   })
+  it("hasccnote returns true if there is a ccnote and the ", () => {
+    const articleInstance = renderer
+      .create(<StandardArticle data={data} pageContext={pageContext} />)
+      .getInstance()
+    let post = { frontmatter: { cconlynote: "testnote" } }
+
+    expect(articleInstance.hasCCNote(post)).toBeTruthy()
+  })
+  it("hasccnote returns false if the roles object does not exist", () => {
+    const articleInstance = renderer
+      .create(<StandardArticle data={data} pageContext={pageContext} />)
+      .getInstance()
+    let post = { frontmatter: {} }
+
+    expect(articleInstance.hasCCNote(post)).toBeFalsy()
+  })
+  it("hasccnote returns false if the environment variable is false", () => {
```

Comment:
If articleInstance.IS_CC_SITE was true, would this test still pass? Setting it to false seems to confuse what the test is testing, as the description says outcome is based on environment variable

Reply:
No it would fail, because there is a cc note - its testing that the environment variable gets caught in the first if block